stl/inc/format

```diff
@@ -3519,7 +3519,10 @@ struct _Format_checker {
     consteval explicit _Format_checker(basic_string_view<_CharT> _Fmt) noexcept
         : _Parse_context(_Fmt, _Num_args), _Parse_funcs{&_Compile_time_parse_format_specs<_Args, _ParseContext>...} {}
     constexpr void _On_text(const _CharT*, const _CharT*) const noexcept {}
-    constexpr void _On_replacement_field(size_t, const _CharT*) const noexcept {}
+    constexpr void _On_replacement_field(const size_t _Id, const _CharT*) const {
+        _ParseContext _Parse_ctx({});

```

Comment:
I feel like this isn't conforming, since this parse context is very observably different from what the standard says we will pass to the user's `parse` function. I am not requesting a change, however, since this is consistent with how we handle simple replacement fields when actually formatting.

Reply:
Agreed. I think #4640 addresses this.